src/Models/JiraWorklog.php

```diff
@@ -0,0 +1,306 @@
+<?php
+/**
+ * Class that pars JIRA worklog data and gives access to it.
```

Comment:
fixed the spelling of pars above but not here ;)

Reply:
lol, fixed!